manifests/profiles/dualstack.yaml

```diff
@@ -0,0 +1,10 @@
+kind: IstioOperator
```

Comment:
Don't you miss the `apiVersion: install.istio.io/v1alpha1` top declaration here?